ArmPkg/Library/ArmLib/AArch64/AArch64ArchTimerSupport.S

```diff
@@ -9,6 +9,41 @@
 
 #include <AsmMacroLib.h>
 
+/* Special accessors for VHE */
+#define cntp_tval_el02          s3_5_c14_c2_0
+#define cntp_ctl_el02           s3_5_c14_c2_1
+#define cntp_cval_el02          s3_5_c14_c2_2
+#define cntv_tval_el02          s3_5_c14_c3_0
+#define cntv_ctl_el02           s3_5_c14_c3_1
+#define cntv_cval_el02          s3_5_c14_c3_2
+
+#define RVHE(reg, sysreg)       rvhe reg, sysreg, sysreg ## 2

```

Comment:
Minor, should we rename the macro names to RCNT as in read counter and  WCNT write counter. The only reason being the macros internally make the decision to call the VHE specific version based on the HCR_EL2.E2H bit value.

Reply:
Fair enough